pyproject.toml

```diff
@@ -90,3 +91,69 @@ disable_error_code = ["arg-type"]
 [[tool.mypy.overrides]]
 module = "data_store/validation/towns_fund/fund_specific_validation/fs_validate_r4"
 disable_error_code = ["arg-type"]
+
+
+# check_untyped_defs enabled exceptions
+# we should get rid of these one by one
```

Comment:
🎉 yay for the incremental approach

Reply:
Proceeds to fix all errors in the next PR: +1000 -1000 changes